tests/devices/test_default_qubit_broadcasting.py

```diff
@@ -2023,7 +2025,7 @@ def circuit():
 
         circuit()
         # evaluated one expval altogether
-        assert spy.call_count == 1
+        assert spy.call_count == 2
```

Comment:
Can you explain what made the behaviour change here? I guess now the tape is split, but it was not the case before?